applications/crossbar/priv/couchdb/schemas/account_rate_limits.json

```diff
@@ -0,0 +1,137 @@
+{
+  "_id": "account_rate_limits",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": true,
+  "name": "Rate limits",
+  "description": "describe limits for SIP requests",
+  "additionalProperties": false,
+  "properties": {
+    "account": {
+      "type": "object",
+      "required": true,
+      "name": "Rate limits",
+      "description": "describe limits for SIP requests",
+      "additionalProperties": false,
+      "properties": {
+        "per_minute": {
+          "type": "object",
+          "required": true,
+          "name": "Per minute",
+          "description": "The limits to apply by a minute",
+          "additionalProperties": false,
+          "properties": {
+            "invites": {
```

Comment:
Take a look at the work to use nestable schemas. You could create a schema for the properties of "per_minute" and "per_second" and include them here and use the "$ref":"sub-schema-name". See the "metaflows" key in the `accounts` schema.